### PR TITLE
Add crowdin file

### PR DESCRIPTION
**Pull request by daniserejo**
#### What problem is this solving?

Add crowdin file

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- `Crowdin file` for crowdin integration
```
Comment (camilavcoutinho) [inline CHANGELOG.md:11]:
```suggestion
- Crowdin file for crowdin integration
```

**Review by camilavcoutinho — APPROVED** [review]

**Review by erickmoreno — APPROVED** [review]
LGTM

**Comment by erickmoreno** [comment]
Hi @camilavcoutinho and @daniserejo,

I already reviewed and approved this PR, however, this check must pass: vtex-io/docs-outdated-select

I cannot merge your code before that validation. Do you guys know how to report that expected status?

**Comment by daniserejo** [comment]
> 
> 
> Hi @camilavcoutinho and @daniserejo,
> 
> I already reviewed and approved this PR, however, this check must pass: vtex-io/docs-outdated-select
> 
> I cannot merge your code before that validation. Do you guys know how to report that expected status?

Done

**Pull request merged**